test/fixtures/comments.yml

```diff
@@ -95,3 +95,11 @@ question_callout:
   comment: Hey, @bob nice question
   timestamp: <%= Time.now.to_i + 11 %>
   thread: /03
+
+comment_tag:
+  uid: 1
+  nid: 8
+  status: 1
+  comment: 'comment with tag #ballom'
```

Comment:
Note that this says `#ballom` but the tag you're looking for below is `#ballon`.